Formula/crowdin.rb

```diff
@@ -1,8 +1,8 @@
 class Crowdin < Formula
   desc "Command-line tool that allows to manage your resources with crowdin.com"
   homepage "https://support.crowdin.com/cli-tool/"
-  url "https://crowdin.com/downloads/crowdin-cli-2.0.16.zip"
-  sha256 "0545c969e22bcbb55a76d39db1650544b9d1912be04226cac0a18fd049b1a238"
+  url "https://downloads.crowdin.com/cli/v2/crowdin-cli.zip"
```

Comment:
This will be parsed as version "cli" not "2.0.17"

We need to use `https://crowdin.com/downloads/crowdin-cli-2.0.17.zip`